docs/meetups/wtd-community-organizing.rst

```diff
@@ -0,0 +1,149 @@
+Volunteer to become a Write the Docs Community Organizer!
+=========================================================
+
+We are connecting the global Write the Docs (WTD) community by hosting local meetup and similar community events near you.
+
+Who are Write the Docs community organizers?
+--------------------------------------------
+
+Write the Docs communinity organizers are volunteers who build and support WTD communities around the world. Their goal is to create a space for documentarians to build relationships and support each other in their local, or time zone based, communities.
+
+Interested in volunteering as an organizer? Read more and `apply today <https://forms.gle/Jx9CniuwEcLrEYPJ7>`__!
+
+Write the Docs Meetups and similar communities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Write the Docs brings together local documentarians and tech enthusiasts through our network of more than 20 meetup groups as well as similar community sites.
+
+Our meetup groups and community events are welcoming spaces for you to learn, connect, and grow in your journey as a documentarian.
+
+
+The path to being an organizer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All of our organizers are volunteers local to the area, or time zone they serve.
+
+1. Determine if you are interested and up to the task
+2. Complete the volunteer application
+3. Join an introductory call with the Meetups and Community Coordinator
+4. Onboard to to Meetup, or similarly agreed community platform, and start organizing events!
+
+If you are interested in volunteering with an established meetup group, fill out `volunteer form <https://forms.gle/DG8FagKdNgAFtEb47>`__. You will follow a similar process that includes an introduction to that particular organizing team.
+
+What's in it for you?
+^^^^^^^^^^^^^^^^^^^^^
+
+Community organizers have access to:
+
+- Discounts for conference tickets
+- Organizer-only slack channel
+- Write the Docs Zoom account for hosting virtual events
+
+Coming soon:
+
+- Community Organizer office hour hosted by the Meetups and Community Coordinator
+- Templates
```

Comment:
Which templates? For what?

Reply:
I'm collecting event and communication templates to help organizers. This allows them to copy and personalize basic event formats, introduction/CoC slides, etc.